ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -22,7 +24,11 @@ type ReduxMappedProps = {
   logoutError: Option<string>;
 };
 
-export type Props = ReduxMappedProps & ReduxProps;
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+export type Props = OwnProps & ReduxMappedProps & ReduxProps;
```

Comment:
This probably doesn't need to be exported (it doesn't seem to be imported anywhere else)